js/public/bindings.js

```diff
@@ -69,60 +73,100 @@ const addBinding = values => {
     }
     // Basically a form but hush
     const binding = document.createElement('div')
-    binding.className = "binding row"
+    binding.className = 'binding row'
     // Select for input type
-    const bindingType = labelInput({ label: "Type", name: 'inputType', elem: 'select', onChange: setBinding })
+    const bindingType = labelInput({
+        label: 'Type',
+        name: 'inputType',
+        elem: 'select',
+        onChange: setBinding
+    })
     binding.appendChild(bindingType)
     // Option: axis
-    const optionAnalog = labelInput({ elem: 'option', value: 'axes', innerText: "Axis" })
-    bindingType.firstElementChild.appendChild(optionAnalog)
+    const optionAnalog = labelInput({
+        elem: 'option',
+        value: 'axes',
+        innerText: 'Axis'
+    })
+    bindingType?.firstElementChild?.appendChild(optionAnalog)
```

Comment:
bindingType is defined just above, it must be a valid element.
Otherwise, we have bigger issues, and don't want to overlook it / safecheck it. Sometimes it's better to crash than not.